README.md

```diff
@@ -1,42 +1,36 @@
-# Knative Client
+# Kn
 
-This section outlines best practices for the Knative developer experience. It is a reference for Knative CLI implementation, and a reference for Knative client libraries.
+The Knative client `kn` is your door to the [Knative](https://knative.dev) world.
+It allows you to create Knative resources interactively from the command line or from within Shell scripts.
 
-The goals of the Knative Client are to:
+`kn` offers you ...
 
-1. Follow the Knative [serving](https://github.com/knative/serving) and [eventing](https://github.com/knative/eventing) APIs
-2. Be scriptable to allow users to create different Knative workflows
-3. Expose useful Golang packages to allow integration into other programs or CLIs or plugins
-4. Use consistent verbs, nouns, and flags for various commands
-5. Be easily extended via a plugin mechanism (similar to `kubectl`) to allow for experimentation and customization
+* ... full support for managing all features of [Knative Serving](https://github.com/knative/serving) (services, revisions, traffic splits)
```

Comment:
Some `Serving` start with an uppercase letter, some do not. Please use a uniform way. So does the word `Eventing`

Reply:
So the convention I used here was

* "Knative Serving" and "Knative Eventing" when referring to thos top-level components of "Knative" as a product/project
* `Service`, `Revision`, `Route` when referring to resources in a literal sense and concrete context.
* "services", "revisions"m "trafifc splits" when used in a more abstract and figurative meaning like here. The boundary to the resources is fluent so.

Does this make sense ?

Reply:
Yes. Thank you, Roland